ledger/src/main/kotlin/net/corda/v5/ledger/notary/NotaryService.kt

```diff
@@ -39,7 +39,7 @@ abstract class NotaryService : SingletonSerializeAsToken {
      * Produces a notary service flow which has the corresponding sends and receives as [NotaryClientFlow][net.corda.systemflows.NotaryClientFlow].
      * @param otherPartySession client [Party] making the request
      */
-    abstract fun createServiceFlow(otherPartySession: FlowSession): Flow<Void?>
+    abstract fun createServiceFlow(otherPartySession: FlowSession): ResponderFlow
```

Comment:
This shouldn't exist as contract for notary should stop at the client side which we factory up based upon advertised notary type in MGM. We should not be requiring that anything is a flow in their protocol. Only contract should be that we can get back a signature if OK, else an error of not valid. The details of transport to and from notary should be absolutely opaque. Ultimately we need to talk to @relyafi about this

Reply:
This change is purely to keep things compiling. The ledger API needs an overhaul along the lines of the flow API (this is why the API is not going to be stable for some time).

Reply:
Yep, I suspect this is simply down to the fact that we're still using the old C4 code in the new C5 repos - this code doesn't exist at all in our DP1 based dev branches, so should be removed altogether when we come to port it across to the new repos.